src/natcap/invest/habitat_quality.py

```diff
@@ -811,9 +813,12 @@ def trim_op(base, cover_x):
         Returns:
             _OUT_NODATA where either array has nodata, otherwise cover_x.
         """
-        return numpy.where(
-            (base == base_nodata) | (cover_x == lulc_nodata),
-            base_nodata, cover_x)
+        result_array = numpy.full(base.shape, base_nodata)
```

Comment:
Good catch! Since the valid values returned are from `cover_x`, I used that as the base for `numpy.full` and set the nodata value to what it says it should return.